core/roboconf-target-docker/src/main/java/net/roboconf/target/docker/internal/DockerHandler.java

```diff
@@ -117,7 +117,7 @@ else if(( image = DockerUtils.findImageByTag( imageId, images )) != null )
 			InputStream response = null;
 			File dockerfile = null;
 			try {
-				dockerfile = new DockerfileGenerator(pack, targetProperties.get(AGENT_JRE_AND_PACKAGES)).generateDockerfile();
+				dockerfile = new DockerFileGenerator(pack, targetProperties.get(AGENT_JRE_AND_PACKAGES)).generateDockerfile();
```

Comment:
"DockerfileGenerator" was intentional (what we generate is called a "Dockerfile", not a "docker file").